styleguide/components/41-organisms/filter/filter.twig

```diff
@@ -1,6 +1,8 @@
 <div class="sidebar-layout filter">
   {% set modalContent %}
     <h2>Filter the results</h2>
+    <p class="help-text">When selecting or deselecting filters, the page is automatically reloaded.</p>
```

Comment:
Best check with @mattiasberlamont if this is the default behaviour (styleguide) or a drupal-exception.
Either way "page reloaded" will only apply to drupal, others will only load the results with ajax (or wait for a submit).

Reply:
@delrueba I think we spoke about this and concluded that the default behaviour (styleguide) should be like a form with one submit button, and thus without disclaimer. Correct? Or would you choose another default behaviour?

Reply:
Agreed 🙂decision was made before my arrival here, so must've been a misunderstanding from my side. I'll remove it from styleguide and add it in gent-base